app/assets/javascripts/modules/all.mjs

```diff
@@ -0,0 +1,32 @@
+// JS Module used to combine all the JS modules used in the application into a single entry point,
+// a bit like `app/__init__` in the Flask app.
+//
+// When processed by a bundler, this is turned into a Immediately Invoked Function Expression (IIFE)
+// The IIFE format allows it to run in browsers that don't support JS Modules.
+//
+// Exported items will be added to the window.GOVUK namespace.
+// For example, `export { Frontend }` will assign `Frontend` to `window.Frontend`
+import Button from 'govuk-frontend/govuk/components/button/button';
+
+// Copy of the initAll function from https://github.com/alphagov/govuk-frontend/blob/v2.13.0/src/all.js
```

Comment:
Version number still old

Reply:
This code is now in `app/assets/javascripts/govuk-frontend-local.mjs` so I've updated it there.